qiskit/aqua/utils/run_circuits.py

```diff
@@ -195,28 +205,33 @@ def _safe_get_job_status(job, job_id):
     return job_status
 
 
-def run_qobj(qobj, backend, qjob_config=None, backend_options=None,
-             noise_config=None, skip_qobj_validation=False, job_callback=None):
+def run_qobj(qobj: QasmQobj,
+             backend: BaseBackend,
```

Comment:
I believe this should be `Union[Backend, BaseBackend]` to support the V2 provider interface (also in the other places where `BaseBackend` is a type hint 🙂)

Reply:
I modified the type hints